docs/contributing/supporting-zulip.md

```diff
@@ -0,0 +1,74 @@
+# Supporting the Zulip project
+
+Some of the most valuable ways to contribute
+don't require touching the codebase at all. For example, you can:
+
+- Report issues, including both [feature
+  requests](../contributing/suggesting-features.md)
+  and [bug
+  reports](../contributing/reporting-bugs.md).
+- [Give feedback](#user-feedback) if you are evaluating or using Zulip.
+- [Participate
+  thoughtfully](../contributing/design-discussions.md)
+  in design discussions.
+- [Sponsor Zulip](https://github.com/sponsors/zulip) through the GitHub sponsors program.
+- [Translate](../translating/translating.md)
+  Zulip into your language.
+- [Stay connected](#stay-connected) with Zulip, and [help others
+  find us](#help-others-find-zulip).
+
+## User feedback
+
+Nearly every feature we develop starts with a user request. If you are part
+of a group that is either using or considering using Zulip, we would love to
+hear about your experience with the product. If you're not sure what to
+write, here are some questions we're always very curious to know the answer
+to:
+
+- Evaluation: What is the process by which your organization chose or will
+  choose a group chat product?
+- Pros and cons: What are the pros and cons of Zulip for your organization,
+  and the pros and cons of other products you are evaluating?
+- Features: What are the features that are most important for your
+  organization? In the best-case scenario, what would your chat solution do
+  for you?
+- Onboarding: If you remember it, what was your impression during your first
+  few minutes of using Zulip? What did you notice, and how did you feel? Was
+  there anything that stood out to you as confusing, or broken, or great?
+- Organization: What does your organization do? How big is the organization?
+  A link to your organization's website?
+
+You can contact us in the [#feedback channel of the Zulip development
+community](https://chat.zulip.org/#narrow/channel/137-feedback) or
+by emailing [support@zulip.com](mailto:support@zulip.com).
+
+## Stay connected
+
+Even if you are not logging into the development community on a regular basis,
+you can still stay connected with the project.
+
+- Follow us [on Twitter](https://twitter.com/zulip).
+- Subscribe to [our blog](https://blog.zulip.org/).
+- Join or follow the project [on LinkedIn](https://www.linkedin.com/company/zulip-project/).
+
+## Help others find Zulip
+
+Here are some ways you can help others find Zulip:
```

Comment:
I think https://zulip.com/help/support-zulip-project might have a better/more modern version of this content? It might be we can delete this page and instead link to that.